mods/seasonal/moves.js

```diff
@@ -1199,6 +1276,38 @@ let BattleMovedex = {
 		target: "self",
 		type: "Bug",
 	},
+	// Hubriz
+	flowertornado: {
+		accuracy: 90,
+		basePower: 95,
+		category: "Special",
+		desc: "",
+		shortDesc: "",
+		id: "flowertornado",
+		name: "Flower Tornado",
+		isNonstandard: true,
+		pp: 10,
+		priority: 0,
+		flags: {protect: 1, mirror: 1},
+		onPrepareHit: function (target, source) {
+			this.attrLastMove('[still]');
+			this.add('-anim', source, "Petal Blizzard", target);
+			this.add('-anim', source, "Leaf Tornado", target);
+		},
+		secondary: {
+			chance: 20,
+			onHit: function (target, source) {
```

Comment:
This works right? I'm p sure it does but just checking that its tested.

Reply:
Yes